resources/pushHandler.js

```diff
@@ -271,9 +271,7 @@ const uploadToContainer = (z, bundle, result) => {
       .then(fileContents => {
 
         //Too many files if a batch was set up, throw an error
-        if(fileContents.length > 50){
-          throw new Error(messages.TOO_MANY_FILES);
-        }
+        fileCountChecker(fileContents.length);
```

Comment:
Is `fileContents` an array or an object?
Looks like it's an array, but you do `Object.keys(fileContents).length` below. This is confusing...

Reply:
It is supposed to be an array, but sometimes when I would push multiple files at once, this condition wouldn't be true. It would go into the single file push process and fail. I'm not sure why this is happening, this just temporarily fixed that.

Reply:
Fixed it, there was some logic wrong in the fileHandler.js file that caused the length to be 0.